conf/machine/mz7010-iocc.conf

```diff
@@ -0,0 +1,39 @@
+#@TYPE: Machine
+#@NAME: mz7010-iocc
```

Comment:
I'm just wondering whether move most of the features into a common Microzed conf file (microzed-common.inc), As this looks on the surface to be exactly the same file for each of the specific Microzed machines

Then use a require to the common file for in each specific board.